spring-integration-zip/src/main/java/org/springframework/integration/zip/splitter/UnZipResultSplitter.java

```diff
@@ -16,15 +16,16 @@
 
 package org.springframework.integration.zip.splitter;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
 import org.apache.commons.io.FilenameUtils;
 import org.springframework.integration.file.FileHeaders;
 import org.springframework.integration.support.MessageBuilder;
 import org.springframework.integration.zip.ZipHeaders;
 import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
```

Comment:
OK. Will do if you find it annouying just as comment from here. The code convention is not so big deal as I said and that we request you to follow it just in case to save the time. Otherwise you will see such a comment from us in you PRs and they won't be (maybe) merged until addressing the code format too.